src/GRA.Controllers/MissionControl/ProgramsController.cs

```diff
@@ -257,6 +263,11 @@ public async Task<IActionResult> Edit(ProgramDetailViewModel model)
                 ModelState.AddModelError("BadgeUploadImage", $"Image must be one of the following types: {string.Join(", ", ValidImageExtensions)}");
             }
 
+            if (string.IsNullOrWhiteSpace(model.BadgeAltText))
+            {
+                ModelState.AddModelError("BadgeAltText", "The Badge's Alt-Text is required.");
+            }
```

Comment:
The ModelError should only be added if the program already has a badge or one is added.

Reply:
Fixed in commit af9d945713d8c860bbba206721d9861589a3135d